docs/developing/publishing-apps.md

```diff
@@ -0,0 +1,105 @@
+# App publishing guide
+
+After you've completed developing and testing your app's package for Sandstorm, you can publish it in the [Sandstorm App Market](https://apps.sandstorm.io/).
+
+## Verify your identity
+
+### Sign up with [Keybase.io](https://keybase.io)
+
+Currently, Keybase is invite-only. If you need an invite, you can contact [Jade](mailto:jade@sandstorm.io) or [Kenton](mailto:kenton@sandstorm.io). You should connect some of your public identites with your Keybase account, like Twitter and Github.
+
+Sandstorm app authors are verified using a PGP key linked to Keybase. You should get the [prerequisites](https://keybase.io/docs/command_line/prerequisites) and follow [their directions](https://keybase.io/docs/command_line/installation) to get their software set up.
+
+### Sign your app
+
+In order to verify that you are the author of the app in question, you need to sign the following ASCII statement: `I am the author of the Sandstorm.io app with the following ID: <app-id>`, where `<app-id>` is the one from your `sandstorm-pkgdef.capnp` file.
+
+To generate a pgp-signature file using gpg, run a command like this:
+
+`echo -n "I am the author of the Sandstorm.io app with the following ID: <app-id>" |
+     gpg --sign > pgp-signature`
+
+### Export your signature
```

Comment:
To be pedantic, you're not exporting the _signature_, you're exporting a copy of the GPG public key (note: I will use PGP key and GPG key interchangeably, sorry about that) so that it can be bundled with the Sandstorm package.

My goal with this comment is to state the technically accurate truth, and then let you (@ocdtrekkie) figure out a way to phrase it which makes sense to readers.